### PR TITLE
install location /usr/local/kibana-xxxx. symlink inside www

### PR DESCRIPTION
**Pull request by KjellKod**

### CODE REVIEW

File: packaging/kibana.spec [hunk 1]
```diff
@@ -41,13 +41,18 @@ cd %{name}
...
-tar xvf target/%{name}-%{kibana_version}-linux-x64.tar.gz -C %{buildroot}/usr/local/www/probe/
+tar xvf target/%{name}-%{kibana_version}-linux-x64.tar.gz -C %{buildroot}/usr/local/
 cp init/kibana.conf %{buildroot}/etc/init
-%post
 
+%post
+link=/usr/local/www/probe/%{name}-%{kibana_version}-linux-x64
+if [ ! -L $link ]; then
+   ln -s /usr/local/%{name}-%{kibana_version}-linux-x64 $link
+   sudo chown -R  nginx:nginx $link
```
Comment (john-gress) [inline packaging/kibana.spec]: The -R is probably not necessary.

**Comment by john-gress** [comment]
:+1:

**Comment by alexweltman** [comment]
:+1:

**Comment by craig-cogdill** [comment]
:+1:

File: packaging/kibana.spec (hunk 1 above)
Comment (KjellKod) [inline packaging/kibana.spec]: it is needed as demoed. I don't understand why, it just is

**Comment by yannick-polius** [comment]
:+1:

**Pull request merged**